x-pack/platform/plugins/shared/index_management/public/application/services/breadcrumbs.ts

```diff
@@ -56,7 +59,14 @@ class BreadcrumbService {
 
   public setup(setBreadcrumbsHandler: SetBreadcrumbs): void {
     this.setBreadcrumbsHandler = setBreadcrumbsHandler;
-
+    this.breadcrumbs.content = [
+      {
+        text: i18n.translate('xpack.idxMgmt.breadcrumb.contentLabel', {
+          defaultMessage: 'Content',
+        }),
+        href: ``,
```

Comment:
Do we need empty href here? EuiBreadcrump has `href?: string` I suppose we can skip this prop in this case

Reply:
yeah, should remove href.

Reply:
updated in 0972fc71a00bc377e941872cca8e4a5eb99b6b98